libs/@guardian/libs/src/consent-management-platform/types/window.d.ts

```diff
@@ -82,19 +91,26 @@ declare global {
 			ccpa?: {
 				loadPrivacyManagerModal?: (id: number) => void;
 			};
```

Comment:
Can this ever be present in the window anymore?

Reply:
It's added to the sourcepoint(_sp_) config/object in the window. This function continues to be used in the AUS region.